src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Avoid using 'any' type unless it's absolutely necessary. Especially avoid disabling typescript rules for extensive use of 'any'. It may lead to runtime errors which are hard to debug. Try to replace 'any' with a more specific type.

Reply:
But this line was in start template. And  we can send any data to the server. I don't understand on what type it should be changed